modules/order/src/Entity/Order.php

```diff
@@ -51,6 +51,7 @@
  *     "canonical" = "/admin/commerce/orders/{commerce_order}",
  *     "edit-form" = "/admin/commerce/orders/{commerce_order}/edit",
  *     "delete-form" = "/admin/commerce/orders/{commerce_order}/delete",
+ *     "reassign-form" = "/admin/commerce/orders/{commerce_order}/reassign",
```

Comment:
You don't need to define the link template, it won't be used by anything (they're usually used by route providers).